packages/web/package.json

```diff
@@ -46,7 +49,8 @@
   },
   "dependencies": {
     "@csstools/normalize.css": "^12.0.0",
-    "@lmc-eu/spirit-design-tokens": "^0.19.0"
+    "@lmc-eu/spirit-design-tokens": "^0.19.0",
+    "@lmc-eu/spirit-icons": "^0.2.0"
```

Comment:
I think we should add the same information to `packages/icons/README.md` as we have here https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#install. Could you do it, please?